lib/idris-controller.coffee

```diff
@@ -46,6 +47,9 @@ class IdrisController
       @messages.attach()
       @messages.hide()
 
+  stopCompiler: =>
+    @model?.stop()
```

Comment:
Will this trigger one of the process alerts to fire, or does it die without a fight?